### PR TITLE
Qa 80 adding sonarqube as gradle task

### PR DESCRIPTION
**Pull request by kumarvastav**

### CODE REVIEW

File: beekeeper-testing-plugin/build.gradle [hunk 1]
```diff
@@ -17,11 +32,17 @@ pluginBundle {
 
     plugins {
         testingPlugin {
-            id = 'io.beekeeper.gradle.plugins.testing'
+            id = 'io.beekeeper.gradle.plugins.sonarqube'
             displayName = 'Beekeeper Testing Plugin'
             description = 'Bekeeper common settings for testing. Includes Code Coverage.'
             website = 'https://github.com/beekpr/beekeeper-gradle-plugins/blob/master/README.md'
         }
+        sonarqubePlugin {
```
Comment (isacandrei) [inline beekeeper-testing-plugin/build.gradle:40]: Please add this to the readme as well.

**Review by isacandrei — COMMENTED** [review]
Tests are failing.

Do we plan to add this plugin to all the Java microservices?

File: beekeeper-testing-plugin/build.gradle (hunk 1 above)
Comment (kumarvastav) [reply beekeeper-testing-plugin/build.gradle:40]: Yes Andrei, the plan is to move all the non-DSL java services first and rest later.

File: beekeeper-testing-plugin/build.gradle [hunk 2]
```diff
@@ -17,11 +32,17 @@ pluginBundle {
 
     plugins {
         testingPlugin {
-            id = 'io.beekeeper.gradle.plugins.testing'
+            id = 'io.beekeeper.gradle.plugins.sonarqube'
```
[the remark this answers is not in the document]
Comment (kumarvastav) [reply beekeeper-testing-plugin/build.gradle:35]: Ah....copy/paste issue. Fixed :)

**Pull request closed without merging**